tests/foreman/ui/test_errata.py

```diff
@@ -75,6 +78,20 @@ def _install_client_package(client, package, errata_applicability=False):
     return result.status == 0
 
 
+def _remove_client_package(client, package, errata_applicability=False):
+    """Remove a package in virtual machine client.
+
+    :param client: The Virtual machine client.
+    :param package: the package (full version, or name) to remove from virtual machine client.
+    :param errata_applicability: If True, force host to generate errata applicability.
+    :returns: True if a package was removed successfully, False otherwise.
+    """
+    result = client.execute(f'yum remove -y {package}')
+    if errata_applicability:
+        _generate_errata_applicability(client.hostname)
```

Comment:
Unless the host is configured not to upload the package profile on yum transactions (which is not the default and very unlikely), a `yum -y remove` should always lead to an automatic applicability calculation for that host. I'm thinking this explicit call may never be needed?

Reply:
Thanks! the existing `_install_client_package()` also has an `errata_applicability` option.
Would an automatic applicability calculation occur for `yum -y install` as well? If so I can update that method too.

Reply:
Yeah, it should be for any package action.